holoviews/util/__init__.py

```diff
@@ -268,14 +268,15 @@ def _build_completer(cls, element, allowed):
         def fn(cls, spec=None, **kws):
             spec = element if spec is None else '%s.%s' % (element, spec)
             invalid = set(kws.keys()) - set(allowed)
+            prefix = None
             if invalid:
                 try:
-                    cls._options_error(list(invalid)[0], element,
-                                       Store.current_backend, allowed)
+                    cls._options_error(list(invalid)[0], element, None, allowed)
```

Comment:
Does this support passing an explicit backend kwarg? The way it works for ``.options`` is that if you pass in a specific backend it becomes completely strict otherwise it's permissive about options which apply to one backend but not another. So if we want to support that you'd pop ``backend`` from kws above and then replace ``None`` here with the popped backend (defaulting to None).

Reply:
Initially I thought it would be trivial to support this but in the end I needed to do a little refactoring in b43f60e to get this working.